recipes/genologics/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set version="0.3.12.post0" %}
+
+package:
+  name: genologics
+  version: "{{ version }}"
+
+source:
+  fn: genologics-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/b4/04/bd023ff57246cd5724aec7fca289d2da519aa3c96cffac8e3a239f073c1e/genologics-{{ version }}.tar.gz
+  md5: 1812a2f27873c230d5a9bc86419bba8a
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - requests
+
+  run:
+    - python 2.6.*|2.7.*  # [py2k]
```

Comment:
The module is supported by python 3. That's why the following 2 lines - I wanted to specify version ranges for both py2 and py3:
```
    - python 2.6.*|2.7.*  # [py2k]
    - python >=3.4  # [py3k]
```
I also tried one line `python 2.6.*|2.7.*|>=3.4`, but it didn't work properly. Is there a standard way to specify python version range more specifically? I didn't find any examples, probably the way to do is just not to specify it? I can just remove the version ranges.